concepts/Code_status/carePlan_getPatientCode.sql

```diff
@@ -0,0 +1,13 @@
+-- NOT SUPPORTED
+
+DROP MATERIALIZED VIEW IF EXISTS patientCodeStatus;
+CREATE MATERIALIZED VIEW patientCodeStatus AS
+
+SELECT "patientunitstayid", "cplitemoffset", string_agg("cplitemvalue", ', ') AS "cplitemvalue"
```

Comment:
It's difficult to make sense of this query, I guess because it was automatically generated. If we are going to include SQL, then ideally it should be readable. If the plan is to automatically generate it, then perhaps include an option to output SQL in the R Markdown?

Reply:
I have added comments in both the .Rmd and .sql files that tells people how the .sql file was generated (using show_query()) to clarify how this was produced.